inbm/fpm/inb-provision-cloud/main.go

```diff
@@ -426,9 +427,22 @@ func configureTls(templateDir string, caFileName string, cloudProviderName strin
 	return string(jsonBytes), caPath
 }
 
+func removeProxySection(template string) string {
+    proxySection := `"proxy": {
+        "hostname": "{PROXY_HOSTNAME}",
+        "port": {PROXY_PORT}
+    },`
+
+    return strings.Replace(template, proxySection, "", 1)
```

Comment:
Proxy section could change. Might be more robust to scan for the 'proxy' section in the JSON and remove the whole thing. An easier stopgap would be to add error handling in this function to cover the case where we can't find the proxy section.

Reply:
Something like this:

```
import json

# Your JSON string
json_string = '{"key1": "value1", "proxy": {"host": "example.com", "port": 8080}, "key3": "value3"}'

# Load the JSON string into a dictionary
json_dict = json.loads(json_string)

# Remove the 'proxy' key
if 'proxy' in json_dict:
    del json_dict['proxy']

# Convert the dictionary back into a JSON string
new_json_string = json.dumps(json_dict)

print(new_json_string)
```